app/controllers/status_updates/comments_controller.rb

```diff
@@ -0,0 +1,3 @@
+class StatusUpdates::CommentsController < CommentsController
+  PATH_PARENTS = [:status_updates]
```

Comment:
I think for the redirect you want you actually need to override the entire create method.